include/config.h

```diff
@@ -46,6 +47,21 @@ struct sway_mode {
 };
 
 /**
+ * libinput options for input devices
+ */
+struct input_config {
+	char *identifier;
+	int click_method;
+	int drag_lock;
+	int dwt;
+	int middle_emulation;
+	int natural_scroll;
+	int scroll_method;
+	int send_events;
+	int tap;
+};
```

Comment:
>What is the usecase for this?

Well, eventually Wayland clients will be able to grab the mouse (like games), but it'd be nice if we could prevent them from being able to grab certain input devices, if that makes sense.

>That sounds like a good idea. 

Yeah, xinput supports it and I actually use that feature, so I'd like to see that in wlc.